sys/include/ztimer/xtimer_compat.h

```diff
@@ -41,10 +41,18 @@ extern "C" {
  */
 #ifndef DOXYGEN
 
+#define XTIMER_WIDTH    (32)
+#define XTIMER_MASK     (0)
```

Comment:
Won't this conflict with the definition in `board.h`?
(sorry, noticed this too late)